catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Service.js

```diff
@@ -114,19 +109,19 @@ define(function (require) {
 
         },
 
-        makeDisableCall: function(){
+        makeDisableCall: function () {
             var model = this;
             var pid = model.get('id');
             var url = [model.configUrl, "disableConfiguration", pid].join("/");
             if (pid) {
                 return $.ajax({
                     url: url,
                     dataType: 'json'
-                }).done(function(){
+                }).done(function () {
                     model.trigger('disabled');
                     //disabling the model means the PID will be regenerated. This model no longer exists on the server.
                     model.destroy();
-                }).fail(function(){
+                }).fail(function () {
                     new Error('Could not disable configuration ' + pid);
```

Comment:
This was just triggered as changed because of formatting. It doesn't look like anything is being done with that error. The enable call above has the same behavior. Should I write a ticket for this?